test/wasmBenchmarker/benchmark.py

```diff
@@ -81,7 +81,32 @@ def parse_args():
     parser.add_argument("--jit", help="use JIT version of Walrus", action="store_true")
     parser.add_argument("--verbose", help="prints extra informations e.g. paths", action="store_true")
     parser.add_argument("--no-system-emcc", help="Don't use emcc command from the system", action="store_true", default=False)
-    return parser.parse_args()
+    parser.add_argument("--jit-no-reg-alloc", help="Use JIT without register allocation", action="store_true", default=False)
+    parser.add_argument("--interpreter", help="Use interpreter", action="store_true", default=False)
+    parser.add_argument("--compare-jit-to-interpreter", help="Compare JIT results to interpreter", action="store_true", default=False)
+    parser.add_argument("--compare-jit-no-reg-alloc-to-interpreter", help="Compare JIT no reg alloc results to interpreter", action="store_true", default=False)
+    parser.add_argument("--compare-jit-no-reg-alloc-to-jit", help="Compare JIT no reg alloc results to JIT", action="store_true", default=False)
```

Comment:
Would it be a good idea to provide a list for thisd parameter? This way any combination would be supported (with any number of columns).